docs/configuration_file.rst

```diff
@@ -1,22 +1,1167 @@
-==========================
-Configuration file example
-==========================
+  
+==================
+Configuration file
+==================
 
-The configuration files follows the `YAML standard <https://yaml.org/>`_.
+The configuration file contains all information needed to build, run and analyse the results from the FlowNet network model.
+
+name
+====
+
+
+
+
+flownet
+=======
+
+A description of the usage of the various keywords available in relation to the construction of the FlowNet network model. 
```

Comment:
Perhaps there should be a reference to `_config_parser.py` for default parameters? Maybe that is better than writing the  default values of every parameter in this document.

Reply:
I guess we should have a better way of listing default values - more consistent between the different keywords, but I don't think we should ask the user to go into the code to find them. I'm sure we will figure something out.